src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,75 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+
+class CalculatorTest {
+    @ParameterizedTest
+    @CsvSource(value = ["3,4", "1,4", "4,7"])
+    fun minus(a: Long, b: Long) {
+        assertThat(Calculator().minus(a, b) == a - b)
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = ["3,4", "1,4", "4,7"])
+    fun plus(a: Long, b: Long) {
+        assertThat(Calculator().plus(a, b) == a + b)
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = ["3,4", "1,4", "4,7"])
+    fun multiply(a: Long, b: Long) {
+        assertThat(Calculator().multiply(a, b) == a * b)
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = ["3,4", "1,4", "4,7"])
+    fun divide(a: Long, b: Long) {
+        assertThat(Calculator().divide(a, b) == a / b)
+    }
+
+    @Test
+    fun `null input`() {
+        assertThatThrownBy {
+            Calculator().calculate(null)
+        }.isInstanceOf(IllegalArgumentException::class.java)
```

Comment:
```suggestion
        assertThrows<IllegalArgumentException> {
            Calculator().calculate(null)
        }
```
좀더 간결한 형태로 타입을 넘길 수 있습니다. 이런 사용법을 알게 되는것을 넘어서 내부적으로 어떻게 가능한지도 알아 보실 수 있는 기회를 가지시면 좋겠습니다. 😀